client/src/components/atoms/RichTextField.tsx

```diff
@@ -155,7 +164,7 @@ const RichTextField: FunctionComponent<Props> = (props: Props) => {
             </div>
             <div className="richtext-field-input">
                 {/* Do not display default text if there is initial content and no change is made */}
-                {!active && !(props.initialContent && !changeMade) &&
+                {changeMade && !active && !props.initialContent && empty &&
```

Comment:
Would it be possible to keep `!(props.initialContent && !changeMade)`? It matches the comment on line 166